pyaedt/modeler/circuits/object3dcircuit.py

```diff
@@ -681,7 +682,7 @@ def angle(self):
                 if "Angle=" in info:
                     self._angle = float(info[6:])
                     break
-        else:
+        elif settings.aedt_version > "2023.2":
```

Comment:
Since version 2023.2 and below are broken in GRPC for `angle`, would it make sense to log a warning when `        settings.aedt_version <= "2023.2"` ?